peregrine/include/generateL2CMcode.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (C) 2016 Swift Navigation Inc.
+#
+# Contact: Dmitry Tatarinov <dmitry.tatarinov@exafore.com>
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+import numpy as np
+
+def generateL2CMcode(PRN):
+  '''
+  The function generates PRN sequence for a particular SV.
+  In the sequence '0' is represented as '1', 1 as -1
+
+  INPUT: SV number from 0 (SV1) to 31 (SV32)
+  OUTPUT: 
+    - PRN seequence array of 10230, 
+    - end state of shift register for testing purpuses
+  '''
+  #--- Sanity sheck for PRN number ------------------------------------------
+  if PRN < 0 or PRN > 31:
+    print 'PRN number(',PRN,') is not in range [0..31]'
```

Comment:
Would be better to do `raise ValueError('PRN number(',PRN,') is not in range [0..31]')` here. This is more typical than returning an error code in Python.

See https://docs.python.org/2/library/exceptions.html.